yuca/ca/neural.py

```diff
@@ -299,7 +310,8 @@ def get_params(self):
         for hh, param in enumerate(self.weights_layer.named_parameters()):
             params = np.append(params, param[1].detach().cpu().numpy().ravel())
 
-        params = np.append(params, self.kernel_params)
+        if self.kernel_params is not None:
```

Comment:
a better fix in future will be to stop having multiple copies of things (e.g. `kernel_params` as a class variable and those same params in `neighborhood_kernel_config` in the `ca_config`

* TODO